lib/bolt/node/ssh.rb

```diff
@@ -127,14 +127,14 @@ def execute(command, sudoable: false, **options)
             unless use_sudo && handled_sudo(channel, data)
               result_output.stdout << data
             end
-            @logger.debug { "stdout: #{data}" }
+            @logger.info { "stdout: #{JSON.dump(data)}" }
           end
 
           channel.on_extended_data do |_, _, data|
             unless use_sudo && handled_sudo(channel, data)
               result_output.stderr << data
             end
-            @logger.debug { "stderr: #{data}" }
+            @logger.info { "stderr: #{JSON.dump(data)}" }
```

Comment:
These are subcommands and should continue to be logged at debug. For example chmoding a task before running it appears here

Reply:
Should this still print the data as JSON?